test/unit/lib/azureMapper.js

```diff
@@ -0,0 +1,88 @@
+const moment = require('moment-timezone');
+const chai = require('chai');
+
+const azMapper = require('../../../app/lib/azMapper');
+const asSampleResponse = require('../../resources/organisations/FK276-as');
+const esSampleResponse = require('../../resources/organisations/FK276-es');
+
+const expect = chai.expect;
+
+function removePastAdditionalDates(esSampleResponseOriginal) {
+  // this needs doing since AS does not return past additional dates whilst ES did
+  // it should make no functional difference to the behaviour
+  const esSampleResponseCopy = JSON.parse(JSON.stringify(esSampleResponseOriginal));
+  const dates = Object.keys(esSampleResponseCopy.openingTimes.alterations);
+  const pastDates = dates.filter(d => moment(d).isBefore('2019-07-31'));
+  pastDates.forEach((d) => {
```

Comment:
The condition for the `filter` could be checked within the `forEach` which would avoid the second (most likely only) partial iteration of the alterations dates. It probably isn't going to be a large performance improvement but I think it would be worth doing e.g.
```js
  Object.keys(esSampleResponseCopy.openingTimes.alterations).forEach((alteration) => {
    if (moment(alteration).isBefore('2019-07-31')) {
      delete esSampleResponseCopy.openingTimes.alterations[alteration];
    }
  });
```

Reply:
Removed to double iteration but retained the explanatory variable